hazelcast/src/main/java/com/hazelcast/map/impl/operation/MultipleEntryBackupOperation.java

```diff
@@ -47,7 +47,7 @@ public void run() throws Exception {
             if (keyNotOwnedByThisPartition(dataKey)) {
                 continue;
             }
-            final Object oldValue = getValueFor(dataKey, now);
+            final Object oldValue = recordStore.get(dataKey, true);
```

Comment:
The problem with true/false as argument is that it doesn't make any sense when you read it unless you look at the get(key,boolean) method. Probably it is better to create 2 constants so that it is obvious what true/false mean